wom/src/test/scala/wdl4s/wom/graph/ScatterNodeSpec.scala

```diff
@@ -65,6 +66,7 @@ class ScatterNodeSpec extends FlatSpec with Matchers {
       ScatterNodeWithInputs(scatterNode, unsuppliedScatterInputNodes) = scatterNodeWithInputs
 
       _ = unsuppliedScatterInputNodes.size should be(0)
+      _ = scatterNode.outputPorts.foreach(System.out.println(_))
```

Comment:
Remove this if it was for your personal debugging. Or if your really want it for some reason, convert `System.out.println` to just `println`.

Reply:
Sigh... I need some equivalent to git secrets that detects when I stupidly leave printlns in my commits